<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuWidget</class>
 <widget class="QWidget" name="MenuWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>MS Shell Dlg 2</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>CTB - Click The Button</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resouces.qrc">
    <normaloff>:/logo/logo/logo.ico</normaloff>:/logo/logo/logo.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">color:#f8f8f8;
background-color:#181818;</string>
  </property>
  <widget class="QPushButton" name="pushButton_main_start">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>301</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>280</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>320</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Semibold</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#383838;</string>
   </property>
   <property name="text">
    <string>Начать тест</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_main_stat">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Semibold</family>
    </font>
   </property>
   <property name="text">
    <string>Статистика</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Open Sans Semibold</family>
       </font>
      </property>
      <property name="text">
       <string>Время выполнения</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox_main_time">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Open Sans Semibold</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#383838;</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>6</number>
      </property>
      <property name="stepType">
       <enum>QAbstractSpinBox::DefaultStepType</enum>
      </property>
      <property name="displayIntegerBase">
       <number>10</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Open Sans Semibold</family>
       </font>
      </property>
      <property name="text">
       <string>Скорость</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_main_speed">
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Open Sans Semibold</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#383838;</string>
      </property>
      <item>
       <property name="text">
        <string>0.25 per sec</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0.5 per sec</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0.75 per sec</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1 per sec</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget_main_stat">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>470</width>
     <height>340</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>470</width>
     <height>340</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>430</width>
     <height>340</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Semibold</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#f8f8f8;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="autoScrollMargin">
    <number>8</number>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="tabKeyNavigation">
    <bool>false</bool>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>90</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>30</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>35</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <row>
    <property name="text">
     <string>Новая строка</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Новая строка</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Новая строка</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Новая строка</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Новая строка</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Новая строка</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Новая строка</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Новая строка</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Новая строка</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Новая строка</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Имя</string>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Скорость</string>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Время</string>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignLeading|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Попаданий</string>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Попаданий в мин</string>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>7</pointsize>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>301</width>
     <height>49</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Open Sans Semibold</family>
       </font>
      </property>
      <property name="text">
       <string>Ваше имя</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_main_username">
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Open Sans Semibold</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resouces.qrc"/>
 </resources>
 <connections/>
</ui>
